src/bin/casc.rs

```diff
@@ -6,19 +6,37 @@ use selinux_cascade::error::ErrorItem;
 use clap::Parser;
 use std::fs::File;
 use std::io::{Error, ErrorKind, Write};
+use walkdir::WalkDir;
 
 #[derive(Parser, Debug)]
-#[clap(author, version, name = "casc")]
+#[clap(author, version, name = "casc", about)]
 struct Args {
+    /// List of input files to process.  Directories are searched recursively.
     #[clap(required(true))]
     input_file: Vec<String>,
 }
 
 fn main() -> std::io::Result<()> {
     let args = Args::parse();
-    let policies: Vec<&str> = args.input_file.iter().map(|s| s as &str).collect();
+    let policies: Vec<String> = match get_policy_files(args.input_file) {
+        Ok(mut s) => {
+            s.sort_unstable();
+            s
+        }
+        Err(e) => {
+            eprintln!("{}", e);
+            return Err(e);
+        }
+    };
+    if policies.is_empty() {
+        // Files supplied on command line, but no .cas files found
+        return Err(Error::new(
+            ErrorKind::InvalidData,
+            "No policy source files found",
+        ));
+    }
     let mut out_file = File::create("out.cil")?;
-    let res = compile_system_policy(policies);
+    let res = compile_system_policy(policies.iter().map(|s| s as &str).collect());
```

Comment:
We could improve `compile_system_policy()` to take an iterator of `&str`, or even better an iterator of `Path`, instead of a `Vec<&str>`. This would avoid such gymnastics.